cmd/scollector/collectors/aws.go

```diff
@@ -0,0 +1,313 @@
+package collectors
+
+import (
+	"fmt"
+	"time"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"bosun.org/slog"
+
+	"bosun.org/_third_party/github.com/awslabs/aws-sdk-go/aws"
+	"bosun.org/_third_party/github.com/awslabs/aws-sdk-go/gen/cloudwatch"
+	"bosun.org/_third_party/github.com/awslabs/aws-sdk-go/gen/ec2"
+	"bosun.org/_third_party/github.com/awslabs/aws-sdk-go/gen/elb"
+)
+
+const (
+	awsCPU                = "aws.ec2.cpu"
```

Comment:
reverted other two commits and committed 888548d727795c015d8e5d58abada23f7b594ad6 to alphabetize constants